romeo/asset-contract/contracts/clarity-bitcoin-mini.clar

```diff
@@ -2,7 +2,7 @@
 
 ;; it can only check if a txid is part of a burn chain block
 
-(define-constant DEBUG-MODE true)
+(define-constant DEBUG-MODE false)
```

Comment:
Now unit tests fail

Reply:
I'll revert the debug mode change as that's being fixed in a separate PR.